urdfenvs/sensors/full_sensor.py

```diff
@@ -42,6 +42,12 @@ def get_observation_space(self, obstacles: dict, goals: dict):
                     high=np.array([5, 5, 5]),
                     dtype=np.float32,
                 )
+            if "radius" in self._obstacle_mask:
```

Comment:
Why is the radius needed? Can't we use 'size' instead, for a sphere it returns the radius actually.

Reply:
True, we can use size instead.